extensions/cornerstone/src/commandsModule.ts

```diff
@@ -445,6 +446,27 @@ const commandsModule = ({ servicesManager }) => {
       );
       toolGroup.setToolEnabled(ReferenceLinesTool.toolName);
     },
+
+    /**
+     * Toggle Image Overlays
+     *
+     * @param param0
+     */
+    toggleImageOverlay: ({ toggledState }) => {
+      toggledState = !toggledState; // revert the state because unlike other toggle tools, this starts as enabled
+
+      const { activeViewportIndex } = ViewportGridService.getState();
+      const viewportInfo = CornerstoneViewportService.getViewportInfoByIndex(
+        activeViewportIndex
+      );
+
+      const viewportId = viewportInfo.getViewportId();
+      const toolGroup = ToolGroupService.getToolGroupForViewport(viewportId);
+
+      toolGroup.setToolConfiguration(ImageOverlayViewerTool.toolName, {
+        showOverlays: toggledState,
+      });
```

Comment:
I don't think there is a need for `showOverlays` as option, since the disabled tool will do the job for you. So instead here you can just disabled it, and if toggle on enable it

Reply:
Doing it through disable/enable also makes the overlay off/on for all viewports in the toolgroup which aligns how other tools work

Reply:
@md-prog - can you apply this change to do it through the enable/disable toolgroup manager?

Reply:
Also, make it a generic enable/disable a named tool group - which means it is generally applicable for other tool types if desired.